java/src/jmri/jmrit/beantable/TransitTableAction.java

```diff
@@ -1476,7 +1476,7 @@ private void newActionPressed(ActionEvent e) {
     private final NamedBeanComboBox<Sensor> whenSensorComboBox = new NamedBeanComboBox<>(
         InstanceManager.getDefault(SensorManager.class), null, DisplayOptions.DISPLAYNAME);
     private final JSpinner whenDataSpinnerFloat = new JSpinner(new SpinnerNumberModel(
-        0.0f, 0.0f, 65.0f, 0.5f)); // delay
+        Float.valueOf(0.0f), Float.valueOf(0.0f), Float.valueOf(65.0f), Float.valueOf(0.5f))); // delay
```

Comment:
A suggestion is to add a comment that the conversion to Float is needed and that float doesn't work here.

Otherwise someone else might come later and do the same change and break the code a second time.

Reply:
Thanks for the reviews / suggestion, I've got some unit tests to upload for these classes which will help prevent future borkage.